src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -136,30 +139,31 @@ extern "C" {
 	}
 
 	NODELOOPPLUGIN_DECL int ibmras_monitoring_plugin_init(const char* properties) {
-		return 0;
+       uv_check_init(uv_default_loop(), &check_handle);
```

Comment:
I know a thing or two about libuv but I don't know what "releasing the event loops" means in this context.